src/irmin/memory_stores/irmin_memory_stores.ml

```diff
@@ -0,0 +1,144 @@
+(*
```

Comment:
I suppose that `src/irmin/mem/irmin_mem.ml` will generate conflicting namespaces? :sadpanda:

Reply:
Indeed. If you prefer, we can go with something like:

- `src/irmin/mem/irmin_memory_stores.ml` & `irmin.mem`
- `src/irmin-mem/irmin_mem.ml` & `irmin-mem`

That's just inconsistent in a different way though.